code/FormSchemaController.php

```diff
@@ -103,16 +103,6 @@ public function schema(HTTPRequest $request): HTTPResponse
         return $this->getSchemaResponse($schemaID, $form);
     }
 
-    /**
-     * Check if the current request has a X-Formschema-Request header set.
-     * Used by conditional logic that responds to validation results
-     */
-    protected function getSchemaRequested(): bool
-    {
-        $parts = $this->getRequest()->getHeader(static::SCHEMA_HEADER);
-        return !empty($parts);
-    }
```

Comment:
This has just been moved further down the file so it's not above the public method